specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -1329,6 +1329,72 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/integrationRuntimes/{integrationRuntimeName}/{integrationRuntimeAction}/operationstatuses/{integrationRuntimeOperationId}": {
```

Comment:
Should it be operationstatus and not operationstatuses?

Reply:
API Call Paths
providers/microsoft.synapse/workspaces/felles-prod-synapse-workspace/integrationruntimes/ssis-intergrationruntime-drage/start/**operationstatuses**/b196212a36104cae81bf85b4e694afe5